outscale/data_source_outscale_load_balancer.go

```diff
@@ -209,38 +199,21 @@ func buildOutscaleDataSourceLBFilters(set *schema.Set) oscgo.FiltersLoadBalancer
 		case "load_balancer_name":
 			filters.LoadBalancerNames = &filterValues
 		default:
-			filters.LoadBalancerNames = &filterValues
 			log.Printf("[Debug] Unknown Filter Name: %s. default to 'load_balancer_name'", name)
 		}
 	}
 	return filters
 }
 
-func readLbs(conn *oscgo.APIClient, d *schema.ResourceData) (*oscgo.ReadLoadBalancersResponse, *string, error) {
+func readLbs(conn *oscgo.APIClient, d *schema.ResourceData) (*oscgo.ReadLoadBalancersResponse, error) {
 	return readLbs_(conn, d, schema.TypeString)
 }
 
-func readLbs_(conn *oscgo.APIClient, d *schema.ResourceData, t schema.ValueType) (*oscgo.ReadLoadBalancersResponse, *string, error) {
-	ename, nameOk := d.GetOk("load_balancer_name")
-	filters, filtersOk := d.GetOk("filter")
-	req := oscgo.ReadLoadBalancersRequest{
-		Filters: &oscgo.FiltersLoadBalancer{},
-	}
-
-	if !nameOk && !filtersOk {
-		return nil, nil, fmt.Errorf("One of filters, or load_balancer_name must be assigned")
-	}
-
-	if filtersOk {
+func readLbs_(conn *oscgo.APIClient, d *schema.ResourceData, t schema.ValueType) (*oscgo.ReadLoadBalancersResponse, error) {
+	req := oscgo.ReadLoadBalancersRequest{}
+	if filters, filtersOk := d.GetOk("filter"); filtersOk {
```

Comment:
This is a singular data_source, therefore to avoid matching with multiple data_source user should add
```
filtersilters, filtersOk := d.GetOk("filter")
if !filtersOk {
   return fmt.Errorf("filters must be assigned")
}
```